src/GoogleSignIn.ts

```diff
@@ -243,31 +248,43 @@ class GoogleSignIn implements GoogleOneTapAuthPlugin {
     // buttonParentElem!.style.border = '2px solid red';  // uncomment for debugging
 
     // The visible button width should be the same as the google button width.
-    const buttonElemWidth = await this.waitForElementOffsetWidthNotZero(buttonElem!);
+    const {width: buttonElemWidth, height: buttonElemHeight} = await this.waitForElementOffsetWidthNotZero(buttonElem!);
     assert(() => buttonElemWidth !== 0);
+    assert(() => buttonElemHeight !== 0);
+
+    // Since we can't set the height on the Google sign-in button, we'll just scale everything up, and adjust its width
+    const googleButtonHeight = 40;
```

Comment:
Where is the height comming from?

Reply:
With the "size" in GsiButtonConfiguration not configured when we call `renderSignInButtonUsingGoogleIdentityServicesForWeb()` from `addSignInActionToExistingButtonForWeb()`, the default size is "large". I wasn't able to find any official documentation specifically saying "large" is 40, but from [the "size" documentation](https://developers.google.com/identity/gsi/web/reference/js-reference#size), those with size="large" are height 40, "medium" are 32, and "small" are 20. I also inspected the element to see that it was 40 there as well. I don't think this is ideal to hardcode here, but I didn't immediately see a better answer to this. E.g. a callback containing a reference to the Google button, once rendered.